install/upgrade/update_to_dev.php

```diff
@@ -202,12 +217,4 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
 
    $table = 'glpi_plugin_flyvemdm_mqttupdatequeues';
    $migration->dropKey($table, 'status');
-
-   $policyCategory = new PluginFlyvemdmPolicyCategory();
-   $policyCategory->import([
-      'completename'    => 'Security > USB'
-   ]);
-   $policyCategory->import([
-      'completename'    => 'Phone'
-   ]);
```

Comment:
creation of new policy categories is automatic from the list pf policies now. These lines are useless